Rox/Framework/TwigView.php

```diff
@@ -49,8 +49,10 @@ class TwigView extends AbstractBasePage {
     public function __construct(Router $router) {
         $this->_loader = new Twig_Loader_Filesystem();
         $this->addNamespace('base');
+        $this->addNamespace('start');
```

Comment:
added the start namespace to twig because the base layout is using it for the nav